common/lib/container-definitions/src/error.ts

```diff
@@ -39,7 +39,10 @@ export interface IErrorBase {
      */
     readonly errorType: string;
     readonly message: string;
-    /** Sequence number when error happened */
+    /**
+     * Sequence number when error happened
+     * @deprecated - Was used for logging and is no longer needed due to DataProcessingError/DataCorruptionError
```

Comment:
I'd guess that it's better to mark it deprecated in 0.39 and simply remove it in main

Reply:
I'll add it to a list of 0.39 changes I may make, and will update this PR to just remove it.

Reply:
#6839